plugins/database/hana/hana_test.go

```diff
@@ -22,8 +23,13 @@ func TestHANA_Initialize(t *testing.T) {
 		"connection_url": connURL,
 	}
 
+	initReq := newdbplugin.InitializeRequest{
+		Config:           connectionDetails,
+		VerifyConnection: true,
+	}
+
 	db := new()
-	_, err := db.Init(context.Background(), connectionDetails, true)
+	_, err := db.Initialize(context.Background(), initReq)
```

Comment:
This should also make assertions on the response object to ensure that the config comes back to Vault properly.